sentry/src/main/java/io/sentry/GsonSerializer.java

```diff
@@ -151,34 +151,30 @@ public GsonSerializer(
    * Serialize a SentryEnvelope to a stream Writer (JSON)
    *
    * @param envelope the SentryEnvelope
-   * @param writer the Writer
+   * @param stream the Stream
    * @throws IOException an IOException
    */
   @Override
-  public void serialize(final @NotNull SentryEnvelope envelope, final @NotNull Writer writer)
-      throws Exception {
+  public void serialize(SentryEnvelope envelope, OutputStream stream) throws Exception {
```

Comment:
```suggestion
  public void serialize(final @NotNull SentryEnvelope envelope, final @NotNull OutputStream stream) throws Exception {
```

M: double-check if there are more missing.

Reply:
H: not sure if it's possible to use, as the OutputStream comes from the connection, but `BufferedOutputStream` exists

Reply:
H: I'd rename to `outputStream` as `inputStream` also exists.